precompile/contracts/rewardmanager/contract.go

```diff
@@ -13,8 +13,8 @@ import (
 
 	"github.com/ava-labs/subnet-evm/accounts/abi"
 	"github.com/ava-labs/subnet-evm/constants"
-	"github.com/ava-labs/subnet-evm/precompile"
 	"github.com/ava-labs/subnet-evm/precompile/allowlist"
+	"github.com/ava-labs/subnet-evm/precompile/contract"
```

Comment:
can we remove the import `"embed" ` below this?

Reply:
it's a native golang package moved it to top.